blog/index.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Blog
+---
+
+{% include back-to-top.html %}
+
+<h1>Blog</h1>
+
+<!-- This loops through the paginated posts -->
+<div class="blog-list">
+{% for post in paginator.posts %}
+<div class="blog-list-item">
```

Comment:
I think it may be interesting to have some of the content cut off in the post list description like the Red Hat blog. https://www.redhat.com/en/blog

e.g.
Red Hat has announced the availability of Ansible Automation Platform in AWS Marketplace and Managed Ansible Automation Platform in Azure Marketplace...[read full post](https://www.redhat.com/en/blog/open-source-any-cloud-how-red-hat-brings-greater-choice-cloud-marketplaces)